### PR TITLE
Authentication Done .

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Screenshot (72)
Screenshot (73)

what to check : 

Pages , contexts , services

what not to check :

CSS , package.json , package.lock.json

### CODE REVIEW

File: src/contexts/authContext.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { useState, createContext,  useContext } from "react";
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+
+  const [auth, setAuth] = useState({
+    token: localStorage.getItem("AUTH_TOKEN"),
+    status: localStorage.getItem("AUTH_TOKEN") ? true : false,
+    // username: localStorage.getItem("username") || "",
```
Comment (xii-vi) [inline src/contexts/authContext.js]: remove comments

File: src/pages/Authentication/loginPage.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const signinSubmitHandler = async (user) => {
+    const { encodedToken } = await signinService(user);
+    if (encodedToken !== undefined) {
+      localStorage.setItem("AUTH_TOKEN", JSON.stringify(encodedToken));
+      setAuth(auth => ({
+        ...auth,
+        status: true,
+        token: encodedToken,
+      }));
+    //   navigate("/")
```
Comment (xii-vi) [inline src/pages/Authentication/loginPage.js]: here also remove comment

**Review by xii-vi — APPROVED** [review]
**Folder and Code structure is good**
left a few sanity comments.
Check before merging.

File: src/services/signinService.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const signinService = async ({ email, password }) => {
+  try {
+    const {
+      data
+    } = await axios.post("/api/auth/login", {
+      email: email,
+      password: password,
+    });
+    console.log(data.foundUser)
```
Comment (xii-vi) [inline src/services/signinService.js:11]: remove console

File: src/services/signupService.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  try {
+    const {
+      data
+    } = await axios.post("/api/auth/signup", {
+      email: email,
+      password: password,
+      firstName : firstName , 
+      lastName : lastName
+    });
+    console.log(data)
```
Comment (xii-vi) [inline src/services/signupService.js:13]: here too

**Review by arpit01923 — APPROVED** [review]
- clean UI and code
- good folder structure 
- try absolute paths to avoid ../../

**Pull request merged**